onnxruntime/python/tools/transformers/optimizer.py

```diff
@@ -423,6 +450,16 @@ def _parse_arguments():
     )
     parser.set_defaults(use_gpu=False)
 
+    parser.add_argument(
+        "--provider",
+        required=False,
+        type=str,
+        default=None,
+        action="store_true",
+        help="Execution provider to use if use_gpu.",
+    )
+    parser.set_defaults(use_gpu=False)
```

Comment:
Remove this line (duplicated with line 451).

Reply:
Corrected this, too.